script/install.sh

```diff
@@ -9,14 +9,37 @@ elif [ "${BOXEN_HOME:-}" != "" ] ; then
   prefix=${BOXEN_HOME:-}
 fi
 
-mkdir -p $prefix/bin
-rm -rf $prefix/bin/git-lfs*
+local_install=false
+
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --local)
+      local_install=true
```

Comment:
It doesn't look like this variable is being used.  Perhaps we can remove it here and on line 12?